group/group_info.py

```diff
@@ -21,12 +21,18 @@
 from group.models import Group
 
 from users.models import User
+from utils.user_info import validate_user
 from utils import user_info
 from utils.log_info import get_logger
+from utils.user_info import has_group_ownership
 from django.http import Http404
 
 logger = get_logger()
 
+def can_edit_group(user, group):
```

Comment:
Please give group coowner edit auth, too.
However, only owner and admin can delete group.